Modules/Filtering/LabelMap/include/itkStatisticsLabelMapFilter.h

```diff
@@ -137,9 +137,11 @@ class ITK_TEMPLATE_EXPORT StatisticsLabelMapFilter
   static constexpr unsigned int
   GetDefaultNumberOfBins()
   {
-    return NumericTraits<FeatureImagePixelType>::IsInteger && sizeof(FeatureImagePixelType) <= 2
-             ? 1 << (8 * sizeof(FeatureImagePixelType))
-             : 128;
+    constexpr size_t bitsShift =
+      std::min(8 * sizeof(typename Self::FeatureImagePixelType), 8 * sizeof(unsigned int) - 1);
```

Comment:
Please consider using `CHAR_BIT`, instead of `8` (two times), according to https://clang.llvm.org/extra/clang-tidy/checks/readability-magic-numbers.html

Why do you use `typename Self::FeatureImagePixelType`, instead of simply `FeatureImagePixelType` (as on the next line)?

Reply:
no, yes. done.